README.md

```diff
@@ -254,8 +258,8 @@ Other Style Guides
 
     // good
     const obj = {
+      anakinSkywalker,// keep up with declare order
```

Comment:
i don't think we need the comment, but the re-sorting is cute

Reply:
agree.